tests/test_regex.py

```diff
@@ -16,37 +16,35 @@ def assertMatch(self, string, expected):
         self.assertIsNotNone(match)
         self.assertEqual(match.groupdict(), expected)
 
-    def test_regex(self):
+    def assertMatchIsNone(self, string):
+        self.assertIsNone(re.match(regex, string))
 
+    def test_no_matches(self):
+        self.assertMatchIsNone('')
+        self.assertMatchIsNone('foo')
+
+    def test_matches(self):
         self.assertMatch(
-            '/path/to/package/module.py:18:4: error: No return value expected',
-            {
+            '/path/to/package/module.py:18:4: error: No return value expected', {
                 'error': 'error',
                 'line': '18',
                 'col': '4',
                 'warning': None,
-                'message': 'No return value expected'
```

Comment:
Gotta say I prefer the previous indentation style.

Reply:
hmm.. yeah I probably do too. It was just when I was adding tests the extra lines seemed a little too verbose. Those tests seemed to slurp up too many extras lines. They next time I look at the code I'll probably realise it better the way it was haha.